src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPageviews.tsx

```diff
@@ -0,0 +1,109 @@
+import { color, Sans } from "@artsy/palette"
+import { ArtworkSidebarPageviews_artwork } from "__generated__/ArtworkSidebarPageviews_artwork.graphql"
+import { ArtworkSidebarPageviewsQuery } from "__generated__/ArtworkSidebarPageviewsQuery.graphql"
+import { ContextConsumer } from "Artsy"
+import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
+import React, { Component } from "react"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { PageviewsIcon } from "Styleguide/Elements/icons/Pageviews"
+
+interface Props {
+  artwork: ArtworkSidebarPageviews_artwork
+}
+
+export class ArtworkSidebarPageviews extends Component<Props> {
+  render() {
+    const { artwork } = this.props
+    const { pageviews } = artwork
+
+    // TODO: Figure out why `jest` tests don't play nicely with this icon.
+    if (pageviews) {
+      return (
+        <>
+          {typeof jest === "undefined" && (
```

Comment:
Unfortunately this icon seems to work fine (in the browser), and the SVG looks fine, but doesn't play well with our usual testing, so am leaving out for now.